kubernetes/loculus/values.yaml

```
@@ -1597,8 +1597,8 @@ defaultOrganisms:
         taxon_id: 3052518
         segment_identification:
           method: "align"
-          nextclade_dataset_server: https://raw.githubusercontent.com/nextstrain/nextclade_data/cornelius-cchfv/data_output
-          nextclade_dataset_name: nextstrain/cchfv/linked
+          nextclade_dataset_server: https://raw.githubusercontent.com/anna-parker/nextclade_data/cchfv_updates/data_output
+          nextclade_dataset_name: community/pathoplexus/cchfv
```

To revert before merging
```suggestion
          nextclade_dataset_server: https://raw.githubusercontent.com/nextstrain/nextclade_data/cornelius-cchfv/data_output
          nextclade_dataset_name: nextstrain/cchfv/linked
```